multimodal/src/autogluon/multimodal/utils/object_detection.py

```diff
@@ -514,36 +514,70 @@ def VOCName2Idx(name):
     return VOC_NAME_TO_IDX[name]
 
 
-def get_image_name_num(path):
-    if "/potholes" in path:
-        start_idx = path.rfind("/potholes") + 9
-    else:
-        start_idx = path.rfind("/") + 1
+def get_image_filename(path: str):
+    """
+    Get the filename (without extension) from its path.
+
+    Parameters
+    ----------
+    path
+        The path of image.
+
+    Returns
+    -------
+    The file name of image.
+    """
+    start_idx = path.rfind("/") + 1
     end_idx = path.rindex(".")
-    return int(path[start_idx:end_idx])
+    return path[start_idx:end_idx]
```

Comment:
This will not work on Windows since it's using `\` as a path separators. I suggest just using `pathlib` + replace backslashes to linux format:

```python
from pathlib import Path

def get_image_filename(path: str):
    return Path(s.replace('\\', '/')).stem
```

Reply:
Done.  Thanks!